mcos/observation_simulator.py

```diff
@@ -40,4 +40,18 @@ def simulate(self) -> (np.array, np.array):
         return x.mean(axis=0).reshape(-1, 1), np.cov(x, rowvar=False)
 
 
+class MuCovJackknifeObservationSimulator(AbstractObservationSimulator):
 
+    def __init__(self, mu: np.array, cov: np.array, n_observations: int):
+        self.mu = mu
+        self.cov = cov
+        self.n_observations = n_observations
+
+    def simulate(self) -> (np.array, np.array):
+        x = np.random.multivariate_normal(self.mu.flatten(), self.cov, size=self.n_observations)
+
+        idx = np.arange(len(x))
+        cov_hat = np.sum(np.cov(x[idx!=i], rowvar=False) for i in range(len(x)))/float(len(x) - 1)
```

Comment:
should it be divided by `len(x)-1` or `len(x)`?

Reply:
True, there are still len(x) samples